R/data.R

```diff
@@ -132,7 +132,7 @@
 #'   \item \code{prop_high_predation}: More details at \code{\link[DSMhabitat]{prop_high_predation}}
 #'   \item \code{contact_points}: More details at \code{\link[DSMhabitat]{contact_points}}
 #'   \item \code{..surv_juv_rear_int}: Intercept, Source: calibration (varies by tributary)
-#'   \item \code{.surv_juv_rear_contact_points}: Coefficient for \code{contact_points} variable, Source: TODO - Literature
+#'   \item \code{.surv_juv_rear_contact_points}: Coefficient for \code{contact_points} variable, Source: Derived from the \href{https://nrm.dfg.ca.gov/PAD/view/query.aspx}{Passage Assessment Database (PAD)} maintained by California Department of Fish and Wildlife.
```

Comment:
the input value is derived from PAD, the coefficient is from literature.

Reply:
removed - keep as TODO